lib/name_whisperer.rb

```diff
@@ -21,7 +21,7 @@ def self.xcodeproj_from_podfile(podfile)
 
     def self.search_folders_for_xcodeproj
       ui = Pod::UserInterface
-      xcodeprojects = Pathname.glob('**/*.xcodeproj')
+      xcodeprojects = Pathname.glob('**/*.xcodeproj').reject { |path| path.to_s.end_with?('Pods/Pods.xcodeproj') }
```

Comment:
This check is added because this method is being called from a place it wasn't before, but it seemed OK to me to exclude `Pods/Pods.xcodeproj`, since this is such a ubiquitous file.